lib/travis/build/script/nix.rb

```diff
@@ -39,8 +42,10 @@ def configure
         def setup
           super
 
+          version = config[:nix_version]
```

Comment:
Could you change this into `nix` rather than `nix_version`? That seems more consistent with how other language runtimes are configured.